docs/tbdev_getting_started.ipynb

```diff
@@ -187,16 +174,84 @@
     },
     {
       "cell_type": "code",
+      "execution_count": 0,
       "metadata": {
-        "id": "n2PvxhOkW7vn",
+        "colab": {},
         "colab_type": "code",
-        "colab": {}
+        "id": "n2PvxhOkW7vn"
       },
+      "outputs": [],
       "source": [
         "!tensorboard dev upload --logdir ./logs"
-      ],
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "5QH5k4AUNE27"
+      },
+      "source": [
+        "Note that each experiment you upload has a unique experiment ID.  Even if you start a new upload with the same directory, you will get a new experiment ID. It is possible to look at the list of experiments you have uploaded using the `list` command."
+      ]
+    },
+    {
+      "cell_type": "code",
       "execution_count": 0,
-      "outputs": []
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "C2Pj3RQCNQvP"
+      },
+      "outputs": [],
+      "source": [
+        "!tensorboard dev list"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JcZOGmjQNWk_"
+      },
+      "source": [
+        "To remove an experiment you have uploaded, you may use the `delete` command and specifiy the appropriate expriment_id."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "VSkJTT9rNWJq"
+      },
+      "outputs": [],
+      "source": [
+        "# You must replace YOUR_EXPERIMENT_ID with the value output from the previous\n",
+        "# tensorboard `list` command or `upload` command.  For example\n",
+        "# tensorboard dev delete --experiment_id pQpJNh00RG2Lf1zOe9BrQA\n",
+        "\n",
+        "!tensorboard dev delete --experiment_id YOUR_EXPERIMENT_ID_HERE"
```

Comment:
Btw, this may cause the doc compilation to fail since it will exit with non-zero.